projects/media-viewer/src/lib/store/store-utils.ts

```diff
@@ -1,19 +1,20 @@
 import {Annotation} from '../annotations/annotation-set/annotation-view/annotation.model';
+
 // @dynamic
 export class StoreUtils {
 
-  static generatePageEntities(annotations): {[id: string]: Annotation[]} {
+  static groupByKeyEntities(annotations, key): {[id: string]: Annotation[]} {
```

Comment:
I guess this is no longer annotations and Annotation[], because it's also used for tag entities

Reply:
correct it should be any[]